src/packages/@ncigdc/modern_components/RepoCasesTable/RepoCasesTable.js

```diff
@@ -15,13 +14,20 @@ import RemoveFromRepositoryCaseSetButton from '@ncigdc/modern_components/setButt
 import { theme } from '@ncigdc/theme';
 import withSelectIds from '@ncigdc/utils/withSelectIds';
 
-export const SearchTable = compose(
+export default compose(
+  setDisplayName('RepoCasesTablePresentation'),
   connect(state => ({ tableColumns: state.tableColumns.cases.ids })),
+  branch(
+    ({ viewer }) =>
+      !viewer.repository.cases.hits ||
+      !viewer.repository.cases.hits.edges.length,
+    renderComponent(() => <div>No results found</div>),
+  ),
   withSelectIds,
 )(
   ({
-    relay,
-    hits,
+    variables,
+    viewer: { repository: { cases: { hits } } },
     entityType = 'cases',
     tableColumns,
     filters,
```

Comment:
Seems like filters isn't passed in here? they are need for creating a new set, right now it always creates a set with  out filters.

Reply:
defaultFilters is coming from above, so renaming this